packages/EasyCodingStandard/tests/Yaml/FileLoader/CheckerTolerantYamlFileLoader/ParametersMergeTest.php

```diff
@@ -71,9 +71,9 @@ public function testMainConfigValueOverride(): void
         // mimics user's "easy-config-standard.yml" with own values
         $delegatingLoader->load(__DIR__ . '/ParametersSource/root-config-override.yml');
 
-        $this->assertArraySubset([
-            'cache_directory' => 'new_value',
-        ], $containerBuilder->getParameterBag()->all());
+        $parameters = $containerBuilder->getParameterBag()->all();
+        $this->assertArrayHasKey('cache_directory', $parameters);
+        $this->assertSame('new_value', $parameters['cache_directory']);
```

Comment:
:+1:

Reply:
Ref https://github.com/rectorphp/rector/issues/1025

Reply:
Can this be done better with assertContains or some other method?

Reply:
It depends on the case. I think this specific code cannot be done better than your way.